front/components/agent_builder/hooks/useAgentPreview.ts

```diff
@@ -59,6 +60,12 @@ export function usePreviewAgent() {
     async (
       overrideName?: string
     ): Promise<LightAgentConfigurationType | null> => {
+      // Don't create draft if MCP server views are still loading
+      if (isMCPServerViewsLoading) {
```

Comment:
You cannot create a draft before MCP server views? 🤔

Reply:
We need the MCP server views to submit the form